### PR TITLE
Rails 4

### PR DESCRIPTION
**Pull request by sauloperez**
Providing support for Rails 4.0. Seems to be working as we tested in a Rails 4 vanilla app.
- [x] Check out `has_many` associations. Do they now accept `scope` as well?

### CODE REVIEW

File: immortal.gemspec [hunk 1]
```diff
@@ -16,8 +16,9 @@ Gem::Specification.new do |s|
   s.executables   = `git ls-files -- bin/*`.split("\n").map{ |f| File.basename(f) }
   s.require_paths = ["lib"]
 
-  s.add_dependency 'activerecord', '~> 3.2.0'
+  s.add_dependency 'activerecord', '<= 4.0.6'
```
Comment (franciscoj) [inline immortal.gemspec]: Why `<=`?

Comment (sauloperez) [inline immortal.gemspec]: If we keep a branch for Rails 4 only, it should actually be `~> 4.0`.

**Comment by enricostano** [comment]
:+1:

**Comment by sauloperez** [comment]
*gusanillo* added @franciscoj

**Pull request merged**